### PR TITLE
Server sent events

### PR DESCRIPTION
**Pull request by adam-fowler**
Example demonstrating server sent events.

### CODE REVIEW

File: server-sent-events/Package.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+let package = Package(
+    name: "server_sent_events",
+    platforms: [.macOS(.v15), .iOS(.v18), .tvOS(.v18)],
+    products: [
+        .executable(name: "App", targets: ["App"])
+    ],
+    dependencies: [
+        .package(
+            url: "https://github.com/hummingbird-project/hummingbird.git",
+            branch: "cancel-on-inbound-close2"
```
Comment (adam-fowler) [inline server-sent-events/Package.swift:15]: Remember to revert once cancelOnInboundClose changes are merged

**Comment by Joannis** [comment]
@adam-fowler why do you need to rely on `main` here?

File: server-sent-events/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# ================================
+# Build image
+# ================================
+FROM swift:5.10-jammy as build
```
Comment (Joannis) [inline server-sent-events/Dockerfile:4]: 6.0?

**Review by Joannis — APPROVED** [review]
Super cool example!

**Comment by Joannis** [comment]
Just two changes needed (6.0 and the change to a release once that's ready)

File: server-sent-events/Dockerfile (hunk 2 above)
Comment (adam-fowler) [reply server-sent-events/Dockerfile:4]: Needed as the new inbound close handler requires 6.0

**Pull request merged**